### PR TITLE
Add diffree as lib for download stats

### PR DESCRIPTION
**Pull request by Lirui-Zhao**

### CODE REVIEW

**Review by Wauplin — COMMENTED** [review]
Hi @Lirui-Zhao, thanks for opening this PR! The config looks good to me :) Two comments before merging this:
- Could you move the config up in the list to keep alphabetical order in the libraries definitions?
- Could you merge this PR (https://huggingface.co/LiruiZhao/Diffree/discussions/1) to add `library_name: diffree` to at least 1 model on the Hub. 

Once that's done, we should be good to merge! Thanks in advance :hugs:

**Comment by Lirui-Zhao** [comment]
> Hi @Lirui-Zhao, thanks for opening this PR! The config looks good to me :) Two comments before merging this:
> 
> * Could you move the config up in the list to keep alphabetical order in the libraries definitions?
> * Could you merge this PR (https://huggingface.co/LiruiZhao/Diffree/discussions/1) to add `library_name: diffree` to at least 1 model on the Hub.
> 
> Once that's done, we should be good to merge! Thanks in advance 🤗

Hi @Wauplin , thank you for helping me complete this pull request! 

I have reordered the config alphabetically while merged the 
 PR (https://huggingface.co/LiruiZhao/Diffree/discussions/1) to add library_name on the hub.

File: packages/tasks/src/model-libraries.ts [hunk 1]
```diff
@@ -165,6 +165,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "doctr",
 		repoUrl: "https://github.com/mindee/doctr",
 	},
+	diffree: {
```
[the remark this answers is not in the document]
Comment (Lirui-Zhao) [reply packages/tasks/src/model-libraries.ts]: Hi, thank you for reminding me of this! I have moved it.

**Review by Wauplin — APPROVED** [review]
Thank you!

**Comment by Lirui-Zhao** [comment]
Hi @Wauplin ,

I hope you’re well.

I’m a bit confused because even though the pull request was successfully merged and the `library_name` is included in the metadata, my model repository ([Diffree](https://huggingface.co/LiruiZhao/Diffree)) still shows the following message:

```
Downloads last month
-
Downloads are not tracked for this model
```

Do you have any insights into why this might be happening?

Thank you!

**Comment by Wauplin** [comment]
Hi @Lirui-Zhao, even though this PR has been merged, it has not been deployed to production yet. Expect a 1-2 days delay before it gets integrated in our main repository + gets deployed.

**Comment by Lirui-Zhao** [comment]
Thanks for the info. 

> Hi @Lirui-Zhao, even though this PR has been merged, it has not been deployed to production yet. Expect a 1-2 days delay before it gets integrated in our main repository + gets deployed.

**Pull request merged**